internal/versions/hash-generator/generate_test.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package main
+
+import (
+	"testing"
+
+	"go.uber.org/goleak"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
What is the use of this file currently? Does the TestMain do anything if there are no test?

Reply:
Iirc yes, the code is compiled into the test.